suzieq/engines/pandas/network.py

```diff
@@ -120,7 +121,7 @@ def _find_bond_members(self, namespace: str, hostname: str,
         # get list  of namespaces, hostnames and ifnames to get info for
 
         ifdf = self._get_table_sqobj('interfaces',
```

Comment:
I think it is important to comment the reason why we we set these values of start time and end time.